benches/eth_deploy_code.rs

```diff
@@ -0,0 +1,78 @@
+use criterion::{criterion_group, criterion_main, BatchSize, BenchmarkId, Criterion, Throughput};
+use secp256k1::SecretKey;
+
+mod common;
+
+use common::{address_from_secret_key, create_eth_transaction, deploy_evm, RAW_CALL};
+
+const INITIAL_BALANCE: u64 = 1000;
+const INITIAL_NONCE: u64 = 0;
+const TRANSFER_AMOUNT: u64 = 0;
+
+fn eth_deploy_code_benchmark(c: &mut Criterion) {
+    let mut runner = deploy_evm();
+    let mut rng = rand::thread_rng();
+    let source_account = SecretKey::random(&mut rng);
+    runner.create_address(
+        address_from_secret_key(&source_account),
+        INITIAL_BALANCE.into(),
+        INITIAL_NONCE.into(),
+    );
+    let inputs: Vec<_> = [1, 4, 8, 12, 16]
+        .iter()
+        .copied()
+        .map(|n| {
+            let code_size = 2usize.pow(n);
+            let code: Vec<u8> = std::iter::repeat(0).take(code_size).collect();
```

Comment:
```suggestion
            let code: Vec<u8> = vec![0; code_size];
```

Reply:
Done in https://github.com/aurora-is-near/aurora-engine/pull/41/commits/4fb6f98cad83c948c7f07d741b46426f4a39f48c